larpandora/LArPandoraEventBuilding/LArPandoraShower/Tools/ShowerPandoraSlidingFitTrackFinder_tool.cc

```diff
@@ -119,7 +120,7 @@ namespace ShowerRecoTools{
 
     // ATTN: Expectations here are that the input geometry corresponds to either a single or dual phase LArTPC.  For single phase we expect
     // three views, U, V and either W or Y, for dual phase we expect two views, W and Y.
-    const bool isDualPhase(fGeom->MaxPlanes() == 2);
+    const bool isDualPhase(lar_pandora::LArPandoraHelper::IsDualPhase());
```

Comment:
Is the explicit lar_pandora namespace needed here?

Reply:
Yes, this one is needed.